cta_names_mappings.json

```diff
@@ -645,5 +659,91 @@
     "full_aliases": ["Mespinoza", "Gold Burlap"]
   },
   "RANCOR": {
+  },
+  "BlackSnakeTeam": {
+  },
+  "RedFoxTrot": {
```

Comment:
The markdown file has an Organisation list for this entry - wanted to add that here too?